### ISSUE TITLE
Some pages are not linked from language.html

### ISSUE THREAD

**Issue by JJ** [perl6/doc#2997]
## The problem

This [page](https://docs-stage.p6c.org/language.html) is supposed to be an introduction to everything. But it does not include, for instance, [`about.html`](https://docs.perl6.org/language/about)

## Suggestions

That page should probably link every other page, but if there's a good reason to avoid some, it's probably OK. Anyway, it would be interesting to know which pages are not linked and why.

**Comment by JJ** [comment]
This disappeared later than [this build](https://github.com/perl6docs/perl6docs.github.io/blob/e2e249239fdf10dde462dd6c394a9aa3b2b90a15/language.html), it was in a section called "General reference", as reported also by @tbrowder.

**Comment by uzluisf** [comment]
The page ["Independent Subroutines"](https://docs.raku.org/language/independent-routines) is also missing from [language.html](https://docs.raku.org/language.html), however that might be intentional.

**Comment by JJ** [comment]
No, it's not. But that page is generated, and it needs to be configured. I'll check what's going on.
